### PR TITLE
Added mvn repository support

### PR DESCRIPTION
**Pull request by rlschilperoort**

### CODE REVIEW

File: mvn.go [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+	"encoding/xml"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+	"time"
+)
+
+type MvnResponse struct {
```
Comment (joohoi) [inline mvn.go:14]: This struct can be removed, as pointed out by the linter.

File: mvn.go [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+	ID   string `json:"_id"`
+	Name string `json:"name"`
+	Time struct {
+		Unpublished NpmResponseUnpublished `json:"unpublished"`
+	} `json:"time"`
+}
+
+type MvnResponseUnpublished struct {
```
Comment (joohoi) [inline mvn.go:22]: This struct can be removed, as pointed out by the linter.

**Review by joohoi — CHANGES_REQUESTED** [review]
Oh, this is great! Thanks for the PR! 

There's a few parts of dead code that can be removed (pointed out in the review comments), and I'd also love to have an entry in `CHANGELOG.md` to let people know that the new version now has Maven support.

Can you also make the change to `-help` output ("mvn" value for `-l`) in `README.md` Usage - section?

I'll do a bit of testing still, but this looks really good!

**Comment by rlschilperoort** [comment]
Glad to help; fixed al problems and changed CHANGELOG.md & README.md

First GO project I ever worked on; never to old to learn.

**Review by joohoi — APPROVED** [review]
Good stuff! LGTM, merging! 

Thanks again!

**Pull request merged**